tests/optimal_screen/screen-adafruit-hdmi.dump

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Is it possible to instead use the dump output from `tvservice --dumpedid <filename>` or the output from `edidparser <monitor_dump>` rather than an already semi-parsed version? It would make reaming this information from monitors easier and allow us to test the parsing step as well.